model/project_aliases.go

```diff
@@ -22,6 +22,7 @@ var (
 	gitTagKey      = bsonutil.MustHaveTag(ProjectAlias{}, "GitTag")
 	remotePathKey  = bsonutil.MustHaveTag(ProjectAlias{}, "RemotePath")
 	variantKey     = bsonutil.MustHaveTag(ProjectAlias{}, "Variant")
+	descriptionKey = bsonutil.MustHaveTag(ProjectAlias{}, "Description")
```

Comment:
Can you please add a description of what it looks like when printing it in the cli?

Reply:
sorry, I meant screenshot

Reply:
"bynn was here" is the description added to the output of the patch command. The above is the change using my own tarball, the below is using native evergreen cli (showing un-updated versions won't be affected by the description change).

Reply:
awesome, thank you